### DISCUSSION TITLE
Compatibility of Tailwind Merge with Unocss

### DISCUSSION THREAD

**Discussion by Sajarin-M** [unocss/unocss#2864] [Q&A]
Is there an equivalent package available for tailwind merge with Unocss?
Can I seamlessly use Tailwind Merge with Unocss without any issues? 🤔

**Comment by wmertens** [comment]
As long as you limit yourself to TW-compatible classnames, I don't see why it wouldn't work?

**Reply by Sajarin-M** [reply]
I hope variant groups are possible. Is it a feasible feature to implement?

**Reply by wmertens** [reply]
It is my understanding that variant groups transform the output, so as long as tailwind merge runs after the transforms, it should work.

**Reply by louiss0** [reply]
```sh
npm i @code-fixer-23/cn-efs
```

```ts
import { tailwindCN_EFS } from "@code-fixer-23/cn-efs"

tailwindCN_EFS("bg-gray-500 bg-gray-700")
```

This library is shorter and framework agnostic

**Comment by louiss0** [comment]
> Is there an equivalent package available for tailwind merge with Unocss? Can I seamlessly use Tailwind Merge with Unocss without any issues? 🤔

I created a library to help with this problem try it out I think you will like it. 

``sh
npm i @code-fixer-23/cn-efs
```

```ts
import { tailwindCN_EFS } from "@code-fixer-23/cn-efs"

tailwindCN_EFS("bg-gray-500 bg-gray-700")
```

This library is shorter and framework-agnostic
